### PR TITLE
Add `insecure-bind-address` option for `npm run start:prod`

### PR DESCRIPTION
**Pull request by shu-mutou**
To run `npm run start:prod` in container for development
and to check it from browser, add option for `insecure-bind-add`
into `.npmrc` and pass it `start:prod`.

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 # Config variables can be overridden by running:
-# npm run <script-name> --kubernetes-dashboard:<variable-name>=<path>
+# npm run <script-name> --kubernetes-dashboard:<variable-name>=<value>
 
 kubernetes-dashboard:kubeconfig_dir = ${HOME}/.kube
 kubernetes-dashboard:kubeconfig = ${HOME}/.kube/config
 kubernetes-dashboard:frontend_build_dir = dist/frontend/en
 kubernetes-dashboard:dashboard_binary_path = dist/amd64/dashboard
 kubernetes-dashboard:dashboard_locale_config = dist/amd64/locale_conf.json
+kubernetes-dashboard:insecure_bind_address = 0.0.0.0
```
Comment (floreks) [inline .npmrc]: Use localhost as default bind address here. Binding to all interfaces by default is not secure.

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "start:frontend:https": "node aio/scripts/version.js && ng serve --progress=false --aot --proxy-config aio/https-proxy.conf.json --ssl --host=0.0.0.0",
```
Comment (floreks) [inline package.json]: Could you also change `--host` parameter here and above to use this new variable?

File: package.json [hunk 3]
```diff
@@ -7,13 +7,13 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "start": "concurrently \"npm run start:backend\" \"npm run start:frontend\"",
+    "start": "concurrently \"npm run start:backend\" \"npm run start:frontend --kubernetes-dashboard:bind_address=$npm_package_config_bind_address\"",
     "start:https": "concurrently \"npm run start:backend:https\" \"npm run start:frontend:https\"",
-    "start:frontend": "npm run postversion && ng serve --aot --progress=false --proxy-config aio/proxy.conf.json --host=0.0.0.0",
-    "start:frontend:https": "node aio/scripts/version.js && ng serve --progress=false --aot --proxy-config aio/https-proxy.conf.json --ssl --host=0.0.0.0",
+    "start:frontend": "echo $npm_package_config_bind_address && npm run postversion && ng serve --aot --progress=false --proxy-config aio/proxy.conf.json --host=$npm_package_config_bind_address",
```
Comment (floreks) [inline package.json]: Is this `echo` a debug leftover?

Comment (shu-mutou) [reply package.json]: Yes, it's my mistake. Sorry. I'll remove it soon.

**Pull request merged**